drivers/reset/Kconfig.arm_scmi

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config ARM_SCMI_RESET
```

Comment:
Is it necessary to add extra KConfig just for 1 option?

Reply:
There 2 option and i expect more to be added (if async or notification added)

Reply:
ok